### PR TITLE
Make the demo cluster use it's own logfile directory

### PR DESCRIPTION
**Pull request by andreasscherbaum**
This will no longer spill logfiles from the demo cluster into
the users ~/gpAdminLogs directory. Also it makes it easier to
identify which logfile was created by the last regression test run,

Addresses #523

### CODE REVIEW

**Comment by danielgustafsson** [comment]
I am for this and completely agree that the demo cluster shouldn't be writing to HOME. Does this require changes in Pulse or any Concourse jobs that run ICG (ie, are there jobs that poke logfiles)?

**Comment by andreasscherbaum** [comment]
I'll tell you after I had a Pulse run. So far I try to get it running on my own machine - see my Snippet on Slack. Want to ensure that I catch all the logfiles.

**Comment by andreasscherbaum** [comment]
After building myself a workaround for the isinf() problem, it turns out that gpinitsystem does not honor the -l flag in most of the places. Especially it does not pass this flag to all the other executables it starts ...

**Comment by ascherbaum-pivotal** [comment]
Clean build on Pulse with the patches: http://pulse-releng.greenplum.com/browse/projects/ascherbaum-GPDB-master-opt-l1/builds/70250/

**Comment by ascherbaum-pivotal** [comment]
@danielgustafsson Can you have another look please, and see if you spot something?

File: gpMgmt/bin/gpstop [hunk 1]
```diff
@@ -118,7 +119,7 @@ class GpStop:
                     self._stop_segments()                                
                     self._stop_gpmmon()
                     self._stop_gpsmon()
-                    self._remove_shared_memory()
+                    self._remove_shared_memory() # this creates a new logfile - why?
```
Comment (danielgustafsson) [inline gpMgmt/bin/gpstop]: This should probably be opened as an issue rather than left as a comment which will most likely be forgotten.

File: gpMgmt/bin/gpstart [hunk 2]
```diff
@@ -70,8 +71,9 @@ class GpStart:
         self.interactive=interactive
         self.timeout=timeout
         self.wrapper=wrapper;
-        self.wrapper_args=wrapper_args;
-        self.skip_standby_check=skip_standby_check;
+        self.wrapper_args=wrapper_args
```
Comment (danielgustafsson) [inline gpMgmt/bin/gpstart]: (Nitpicking) Seems like unrelated cleanup of trailing semicolon? If so then might as well do `.wrapper` on the line above too and put them all in a separate commit.

**Comment by danielgustafsson** [comment]
Looks good to me, with Pulse being happy I am definately :+1:

File: gpMgmt/bin/gpstop (hunk 1 above)
Comment (andreasscherbaum) [inline gpMgmt/bin/gpstop]: I plan to send an email to one of the internal lists, in order to figure out what is going on here.
For now this is my marker where to look into the issue, will remove before commit.

File: gpMgmt/bin/gpstart (hunk 2 above)
Comment (andreasscherbaum) [inline gpMgmt/bin/gpstart]: Makes sense, I make this a separate PR (and maybe scan for more)

**Comment by andreasscherbaum** [comment]
Thanks for the review!

File: gpMgmt/bin/gpstop (hunk 1 above)
Comment (danielgustafsson) [inline gpMgmt/bin/gpstop]: Ok, then I understand, thanks.

File: gpMgmt/bin/gpstart (hunk 2 above)
Comment (danielgustafsson) [inline gpMgmt/bin/gpstart]: +1

**Comment by andreasscherbaum** [comment]
Looks like the additional empty logfile is created on the segments - and happen to be on the same host when the demo cluster is run. The logic in pgMgmt is not clear, and not well documented. But this is beyond this PR to fix.

I plan to commit this PR with the empty logfile.
